Sources/MapboxCoreNavigation/NavigationSettings.swift

```diff
@@ -1,6 +1,8 @@
 import Foundation
 import MapboxDirections
 
+public typealias NavigationRouterType = MapboxRoutingProvider.Source
```

Comment:
Document this public type alias.

For consistency with `MapboxRoutingProvider.Source`, call it `RoutingProviderSource` or just write it out in full. Otherwise, `NavigationRouterType` could get confused with `Router.Type`, which various parts of Core Navigation call a “router type”.